apps/site/test/site_web/views/teaser_view_test.exs

```diff
@@ -67,6 +75,22 @@ defmodule SiteWeb.TeaserViewTest do
     end
   end
 
+  describe "teaser_topic/1" do
+    test "returns a non-linked topic" do
+      teaser = %{@dummy_teaser | topic: "Guides"}
+
+      assert "Guides" == teaser_topic(teaser)
+    end
+
+    test "returns a linked topic" do
```

Comment:
I think these test descriptions could be expanded on to make our intent clear — "returns a non-linked topic" and "returns a linked topic" under what conditions?